deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
```

Comment:
Makes sense, thanks for the link.
I've updated the script to set ALLUXIO_UNDERFS_ADDRESS and access keys.
`~/.bashrc` is used to keep the env settings global.